packages/@streamjar/ui-react/src/lib/menu/menu.tsx

```diff
@@ -39,12 +39,10 @@ export class Menu extends React.PureComponent<IMenuProps, IMenuState> {
 
 		this.state = { hide: false };
 		this.menuRef = React.createRef();
-		this.close = this.close.bind(this);
-		this.renderMenu = this.renderMenu.bind(this);
 	}
 
-	public componentWillReceiveProps(prev: IMenuProps, next: IMenuProps): void {
-		if (prev.anchor !== next.anchor) {
+	public componentWillUpdate(prev: IMenuProps): void {
```

Comment:
this method is still deprecated. :P https://reactjs.org/docs/react-component.html#updating

Reply:
fixed in a later PR 
![image](https://user-images.githubusercontent.com/2203091/46105630-7418b480-c1ce-11e8-8280-d523ffba8b28.png)

Reply:
ok